style.css

```diff
@@ -0,0 +1,247 @@
+body {
+  background-color: #eee;
+  font-family: "Lato", sans-serif;
+}
+
+header {
+  position: fixed;
+  min-height: 75px;
+  padding: 15px 20px;
+  display: flex;
+  justify-content: space-around;
+  background-color: #eee;
+  top: 0;
+  align-items: center;
```

Comment:
The align-items will have no effect since you are on the main-axis

Reply:
Because the default axis is the main axis which is what you used giving it a cross-axis property won't work kindly do some adjustment please.